api/Document.json

```diff
@@ -11622,14 +11642,24 @@
             "ie": {
               "version_added": "10"
             },
-            "opera": {
-              "version_added": "12.1",
-              "notes": "Doesn't fire the <code>visibilitychange</code> event when the browser window is minimized, nor when <code>hidden</code> is set to true."
-            },
-            "opera_android": {
-              "version_added": "12.1",
-              "notes": "Doesn't fire the <code>visibilitychange</code> event when the browser window is minimized, nor when <code>hidden</code> is set to true."
-            },
+            "opera": [
+              {
+                "version_added": "12.1"
```

Comment:
If the Chrome data is correct, this should be removed in 15 and added back in 20.

Reply:
I've confirmed using http://software.hixie.ch/utilities/js/live-dom-viewer/?saved=8817 that the unprefixed `visibilityState` was gone in Opera 15, and then back in 20.